client/src/js/components/Auth/SignUp.jsx

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
 import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 import toastr from 'toastr';
 import {
   apiRegisterUser,
   apiLoginUser } from '../../actions/auth';
```

Comment:
I feel this might look cleaner
`
import {
    apiRegisterUser,
    apiLoginUser
} from '../../actions/auth';
`